utils/kubernetes/client.go

```diff
@@ -49,21 +48,25 @@ func DetectKubeConfig(configfile []byte) (config *rest.Config, err error) {
 func processConfig(configFile []byte) ([]byte, error) {
 	cfg, err := clientcmd.Load(configFile)
 	if err != nil {
+
```

Comment:
Bleck. Let's set some standard formatters as a convention and see if we can stick to those as a project. // @layer5io/adapter-maintainers

Reply:
@Revolyssup in the meantime, can you revert you changes to this file?